src/test/data/JsonSerializableAddressBookTest/duplicateStudentAddressBook.json

```diff
@@ -23,5 +23,7 @@
     "thursdayDismissal" : "1500",
     "fridayDismissal" : "1500",
     "id": "1"
-  } ]
+  } ],
+
+  "trainings" : []
```

Comment:
We putting training data in student json file too?

Reply:
cos AddressBook now needs to have a training field, just added it in so the test cases can run

Reply:
@sudogene actually yes, each student will also store the dates of trainings assigned to them, so should be in their json too, we can look at this again after the add student to training command is added